recipes-kernel/linux/linux-mainline_%.bbappend

```diff
@@ -0,0 +1,8 @@
+# Add files directory to build path
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
```

Comment:
@DaniilKl let's add here a comment why we need to do such thing

Reply:
@TomaszAIR Done: https://github.com/3mdeb/meta-readonly-rootfs-overlay/pull/5/commits/14ed23725afa78a0e447b04ab05dc4ec0d450a4a